lib/core/versions/latest/FeeManager.ts

```diff
@@ -17,41 +17,54 @@ export default class FeeManager {
    *
    * @throws if the number of operations are <= 0.
    */
-  public static convertNormalizedFeeToTransactionFee (normalizedFee: number, numberOfOperations: number, feeMarkupFactor: number): number {
+  public static convertNormalizedFeeToTransactionFee (normalizedFee: number, numberOfOperations: number, feeMarkupPercentage: number): number {

```

Comment:
Optional: consider `computeTransactionFee()` as `normalizedFee` is technically just one of 3 parameters used for the computation.